### ISSUE TITLE
/mobile page code

### ISSUE THREAD

**Issue by ejregithub** [mozilla/bedrock#7859]
## Description 

The /firefox/mobile page needs to be update with new copy and design to align with Skyline and product branding

@stephaniehobson 's branch is here to continue - https://github.com/mozilla/bedrock/tree/mobile-rebased

Copy for the page can be found: https://docs.google.com/document/d/1wmZ0sjkxG8oagTU3eQSUENQaAJJSg6lpQFGwTsLyb68/edit?ts=5db71784

You can find a Demo of the on going work here: https://bedrock-demo-errolyn.oregon-b.moz.works/en-US/firefox/mobile/

---

#### :yellow_heart: Success Criteria :yellow_heart:
- [ ] Is live by end of S1/Q4/2019
- [x] Has removed the Focus content at the bottom of the page

#### Tasks
- [x] Add in app store badges, google play the iOS app store
- [ ] In the badges, use direct links to each app store as indicated in #7860
- [x] Remove the Focus content at the bottom of the page

**Comment by Errolyn** [comment]
@ejregithub can the design and copy docs be linked in here? I have the assets from Cristina. I'm a bit confused with the link to Stephanie's PR, is that the only string change, or is there more I will need to do? also that is on /switch "https://www.mozilla.org/en-US/firefox/switch/" and I thought the work I was going to be doing was on /mobile? Sorry if I missed something.

**Comment by ejregithub** [comment]
@stephaniehobson - can you please point to the link with you branch that has /mobile in it?

@Errolyn 
- yes, it's /mobile, sorry for any confusion.
- there's only the one string change of which I'm aware.

**Comment by Errolyn** [comment]
Stephanie linked the branch in slack its: https://github.com/mozilla/bedrock/tree/mobile-rebased

**Comment by ejregithub** [comment]
@Errolyn - this image needs to replace the current one before we can launch.

![fx-device-2](https://user-images.githubusercontent.com/20547886/66961929-99b86c00-f024-11e9-92cc-f9ae30f82ade.png)

**Comment by dzingeek** [comment]
based on some great feedback from @Errolyn we have decided to add an icon to the text cta so it's more clear to the user that it's not your average text link.

attached is the svg for the icon as well as a screenshot for reference. 

![Screen Shot 2019-10-18 at 2 30 16 PM](https://user-images.githubusercontent.com/3740746/67129670-234a7400-f1b4-11e9-85b8-51d0ae3d8f1b.png)

[icon-send-device.svg.zip](https://github.com/mozilla/bedrock/files/3745753/icon-send-device.svg.zip)

**Comment by Errolyn** [comment]
Added :D

**Comment by Errolyn** [comment]
Ok, the demo is up to date with all the changes, I am going to handle the link changes in another pr so this can start getting reviewed.

**Comment by ejregithub** [comment]
@amychurchwell - minor ask from Winston to add a Firefox logo in the modal window.

Also, as stated earlier today, Natalie and Aleks plan on finalizing copy tomorrow, for en-US.  I'll update this issue when that is confirmed as final/approved.

![Screen Shot 2019-10-22 at 11 33 28 AM](https://user-images.githubusercontent.com/20547886/67320605-4e3e0c00-f4c3-11e9-9934-1e33cb8948b4.png)

**Comment by dzingeek** [comment]
@ejregithub and @amychurchwell i don't think we should have the logo next to the words like that. it's not on brand (competes with how we do our logo lockups). i would suggest if we need the logo in the modal it goes above "get firefox for mobile" centered. let me know if you have any questions.

**Comment by ejregithub** [comment]
Sure @dzingeek - @amychurchell - does that work for you? Logo centered above the headline?

**Comment by amychurchwell** [comment]
Logo centered above headline works for me. :+1:

**Comment by ejregithub** [comment]
@Errolyn - copy is now fully approved/final, including Legal review (no changes required). You can, please, proceed with all assets as final.

**Comment by ejregithub** [comment]
@Errolyn - please note that we will update the meta data

**Comment by rraue** [comment]
I reviewed and the newly added schema data to the /mobile page is not on demo.

**Comment by ejregithub** [comment]
@Errolyn ^ I wanted to confirm that you saw @rraue 's comment above?

**Comment by Errolyn** [comment]
I hadn't, thank you for pointing that out!

**Comment by rraue** [comment]
@Errolyn & @ejregithub But the structured data is now in the demo linked in the issue, so everything looks great there.
